portal/src/texts/profile/tone.md

```diff
@@ -1,51 +1,55 @@
 ## Klart språk
 
-Her kan det hende vi må supplere med mer info fra marked/Charlotte etter hvert som det blir klart. For eksempel forslag til gode måter å formulere seg på.
-For deg som designer og utvikler løsninger for Fremtind, er det viktig at du tenker på klarspråk når du skriver tekster som brukerne skal lese. Veiledninger til brukeren i fagsystemer og skjemaer ut mot publikum må også være tydelige og klare. Spesielt viktig med klarspråk er det for eksempel når skriver feilmeldinger og hjelpetekster. En feilmelding skal fortelle tydelig hva som er feil og foreslå en løsning.
+For deg som designer og utvikler løsninger for Fremtind, er det viktig at du tenker på klarspråk når du skriver tekster som brukerne skal lese. Veiledninger i et skjema til kunder eller tekster i et internt fagsystem må være tydelige og klare. 
```

Comment:
Is `klarspråk` itself a clearly defined term that everyone knows? Maybe link out språkrådet for definition and help?

Reply:
Eller enda bedre, forklare kort hva det er?